packages/whats-new/src/whats-new-page.js

```diff
@@ -28,15 +29,19 @@ function WhatsNewPage( {
 	return (
 		<GuidePage className="whats-new-page__container">
 			<div className="whats-new-page__text">
-				{ heading && <h1 className="whats-new-page__heading">{ heading }</h1> }
+				{ heading && <Text variant="title.large">{ heading }</Text> }
 				<div className="whats-new-page__description">
-					{ description && <p>{ description }</p> }
+					{ description && <Text variant="body">{ description }</Text> }
 					{ link && (
-						<p className="whats-new-page__link">
-							<a href={ link } target="_blank" rel="noreferrer">
-								{ __( 'Learn more' ) }
-							</a>
-						</p>
+						<Button
+							className="whats-new-page__link"
+							href={ link }
+							isTertiary
+							isLink
+							target="_blank"
+						>
+							{ __( 'Learn more' ) }
```

Comment:
Sorry I didn't catch this before, but do we need to pass `__i18n_text_domain__` to `__` like we do in other packages?